Source/WebKit/UIProcess/Cocoa/WebPasteboardProxyCocoa.mm

```diff
@@ -633,7 +633,6 @@
     if (!pageID)
         return DataOwnerType::Undefined;
 
-#if HAVE(PASTEBOARD_DATA_OWNER)
```

Comment:
Is this iOS only code (i.e. is there an ifdef somewhere outside the visible part)? `HAVE(PASTEBOARD_DATA_OWNER)` appears to be false on macOS. 

Also, where does macCatalyst fall in terms of having it?

Reply:
Mac Catalyst has it, but let me double check that macOS is safe.

Reply:
With the latest patch, here's how this works: Any platform may call into `dataOwnerForPasteboard()`, and all except for iOS should return `::Undefined` as specified in the PageClient's default implementation. I've updated the commit message to clarify this.

Arguably this is the wrong thing to do as it imposes an iOS-only concept onto all platforms. @whsieh can probably advise if it's better to simply convert all HAVE(PASTEBOARD_DATA_OWNER) blocks into PLATFORM(IOS), which would match existing behavior but would mean less cleanup.